src/com/facebook/buck/jvm/java/BaseCompileToJarStepFactory.java

```diff
@@ -16,19 +16,24 @@
 
 package com.facebook.buck.jvm.java;
 
+import com.facebook.buck.android.AndroidPlatformTarget;
```

Comment:
I'd like to avoid having this Java code relying on anything Android-specific.  Can the required information be passed from above in some generic way?  ("extraEnvironmentForPostprocessing" or something like that?)